### PR TITLE
Created explorer section in template + footnote handling

### PR DESCRIPTION
**Pull request by jessicamindel**
Minimally modified processor to include footnote support using showdown-footnotes. Added .explorer CSS class to handle new show/hide explorer section, to be used for footnote content (envisioned as a way to include supplementary material for curious students who finish lab early or want to learn more on their own time).

screen shot 2019-02-03 at 3 35 01 pm

### CODE REVIEW

**Comment by jessicamindel** [comment]
I can also go ahead and move the drop-down arrow to the right side, if it might match the rest of the document style better.

**Review by thesephist — CHANGES_REQUESTED** [review]
Left some small comments on the CSS section. Looks good otherwise. Thanks for being on top of the game 🚀

File: resources/assets/styles/style.css [hunk 1]
```diff
@@ -166,3 +176,69 @@ details > summary {
...
+
+/* Footnotes (href value comes from showdown-footnotes) */
+a[href^="#footnote"] {
+    text-decoration: none;
+    color: #78c272;
+    transition: color ease 0.2s;
+}
+
+a[href^="#footnote"]:hover {
+    color: #46A03E
```
[the remark this answers is not in the document]
Comment (jessicamindel) [reply resources/assets/styles/style.css]: Oof, negligent on my part--yep, I'll fix that real quick, alongside the other suggested changes. Thank you!

**Comment by jessicamindel** [comment]
Found an issue (credit to @thesephist) with how showdown-footnotes is handling footnotes--it seems to only convert the first inline footnote to a styled link (regardless of index). The rest are ignored, even though the footnotes to which they should be linking are being properly converted and styled.

**Review by thesephist — APPROVED** [review]

**Comment by thesephist** [comment]
For record keeping: the bug we found is a bug in `showdown-footnotes`: https://github.com/Kriegslustig/showdown-footnotes/pull/3/commits/df4986767bd32b39c1c6babc5d2e85cc7d36de2d

It's a single-character change, but I don't think it's worth filling a PR to the original project since it doesn't look maintained. We could fork it and host it under the WDD org on NPM.

**Comment by jessicamindel** [comment]
We could alternatively (if it doesn't break encapsulation or separation of concerns too much) just create a separate module within the project, since it seems like such a small extension. We'd have one fewer dependency.

**Comment by thesephist** [comment]
@jessicamindel I think it's fine to just keep the dependency (I'm like 90% done forking and publishing it already lol). But you have a good point. I feel fishy about taking their code without implicitly giving credit thru the import though.

I'll make a separate PR for pulling a patched version of `showdown-footnotes` into this.

**Comment by jessicamindel** [comment]
Gotcha. Yeah, agreed--credit is important. I did go ahead and try creating the second file, and made a few changes to enable backlinks on the footnotes in the footnotes section--let me know if you want me to send that over.

**Pull request merged**